Lib/fontTools/ttLib/tables/_n_a_m_e.py

```diff
@@ -145,13 +149,49 @@ def setName(self, string, nameID, platformID, platEncID, langID):
 		else:
 			self.names.append(makeName(string, nameID, platformID, platEncID, langID))
 
+	def _findUnusedNameID(self, minNameID=256):
+		"""Finds an unused name id.
+
+		The nameID is assigned in the range between 'minNameID' and 32767 (inclusive),
+		following the last nameID in the name table.
+		"""
+		names = getattr(self, 'names', [])
+		nameID = 1 + max([n.nameID for n in names] + [minNameID - 1])
+		if nameID > 32767:
+			raise ValueError("nameID must be less than 32768")
+		return nameID
+
+	def addMultilingualName(self, names, ttFont, nameID=None):
```

Comment:
I think we still want to take platforms to add.  Ie. for Noto fonts, and most other uses, I believe we don't want to add Mac platform names, as Mac happily parses and uses Windows names.